prime-router/settings/STLTs/AR/AR-phd.yml

```diff
@@ -0,0 +1,66 @@
+# To submit this to localhost, run:
+#
+# ./prime multiple-settings set --env staging --input ./settings/staging/0170-ar-settings.yml
+---
+- name: "ar-phd"
+  description: "Arkansas Public Health Department"
```

Comment:
Should we put the setting as this or as we did before (i.e ./setting/staging/0170-ar-phd-.....)?

Reply:
I'm fine either way. Jamie moved them to this folder when he moved them out of the mappings folder but it doesn't really matter to me.